### ISSUE TITLE
Bad Trakt API key giving 403: Forbidden error.  Need clearer message

### ISSUE THREAD

**Issue by GertJanFrl** [pbrink231/plex_top_playlists#4]
After some hacking to get it working with Python3 where urllib2 isn't one package anymore, I get some other issues.. I can not figure out what the issue might be.. So I hope you could help me..

`Traceback (most recent call last):
  File "plex_playlist_update.py", line 311, in <module>
    list_updater()
  File "plex_playlist_update.py", line 302, in list_updater
    run_movies_lists(plex)
  File "plex_playlist_update.py", line 265, in run_movies_lists
    trakt_weekly_imdb_ids = trakt_watched_imdb_id_list()
  File "plex_playlist_update.py", line 173, in trakt_watched_imdb_id_list
    response = urllib.request.urlopen(request)
  File "/usr/lib/python3.4/urllib/request.py", line 161, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.4/urllib/request.py", line 469, in open
    response = meth(req, response)
  File "/usr/lib/python3.4/urllib/request.py", line 579, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.4/urllib/request.py", line 507, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.4/urllib/request.py", line 441, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.4/urllib/request.py", line 587, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden`

Apparently there is a 403 error (forbidden), might by because I don't have a trakt API key. (Not supported in V2 anymore?)

**Comment by pbrink231** [comment]
I was testing and it looked like I didnt need the API key to get the lists.  I modified the script because of that.
I then through i could leave it blank because i am grabbing general lists.
I think they are caching the request IP address so even though I am running without the key it is already authenticated and allowing me. 

Your situation is due to a bad Trakt API key.  I emailed you mine to your email on here, github.  Let me know if that fixes it for you.

Will also update to check the request to give a better error message in the future.
